testsuite/features/secondary/min_activationkey.feature

```diff
@@ -6,6 +6,7 @@ Feature: Bootstrap a Salt minion via the GUI with an activation key
 
   Scenario: Log in as admin user
     Given I am authorized for the "Admin" section
+    And I am logged in API as user "admin" and password "admin"
```

Comment:
If I can add a nitpick, "logged from API as" to match the "Logout from API" which I think is the grammatically correct. That would require the renaming of the step as well though.

Reply:
Sorry but that's out of scope, those step definitions were created looong time ago by Eric. Let's keep it like they are for this PR.

Reply:
I would agree. E.g.
```suggestion
    And I am logged in via API as user "admin" and password "admin"
```
sound better.

Reply:
Sorry but no, I will not change this step definition in this PR. This is out of the scope.